storage/rocksdb/ha_rocksdb.cc

```diff
@@ -3678,10 +3681,12 @@ class Rdb_snapshot_status : public Rdb_tx_list_walker {
               "LOCK TYPE: %s\n"
               "INDEX NAME: %s\n"
               "TABLE NAME: %s\n",
-              trx_info.trx_id, trx_info.cf_name.c_str(),
+              "Timestamp: %" PRId64 "\n", trx_info.trx_id,
```

Comment:
Same thing here with timestamp being first. Also for consistency, it looks like we should capitalize TIMESTAMP.